dpnp/dpnp_iface_trigonometric.py

```diff
@@ -57,8 +57,13 @@
     dpnp_atanh,
     dpnp_cos,
     dpnp_cosh,
+    dpnp_exp,
+    dpnp_expm1,
     dpnp_hypot,
     dpnp_log,
+    dpnp_log1p,
+    dpnp_log2,
+    dpnp_log10,
```

Comment:
```suggestion
    dpnp_log10,
    dpnp_log2,
```

Reply:
`pre-commit` (`isort (python)`) automatically changes the order to the current one here.

Reply:
I see, then please ignore my comment around here